.github/workflows/cli.yaml

```diff
@@ -17,7 +17,12 @@ permissions:
     packages: write
 jobs:
     build:
+        name: release deb
         runs-on: ubuntu-latest
+        env:
+          MYCLI_DEB_KEY: ${{ secrets.GH_MYCLI_DEB_KEY }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.GH_AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.GH_AWS_SECRET_ACCESS_KEY }}
```

Comment:
Where's the code where we create this credential/role?

It should be very restricted. 

The ideal is to use short-lived tokens, but if it's too complicated and the current role has the restrictions only to do what's needed by the job, I don't see a problem.

Maybe @tuler has another opinion on this.

Reply:
The credential was being created by terraform. But I asked @oap75 to remove the creation of credentials, leaving the creation of a user, and I'd create credentials manually (using console), and copy to GitHub secret.

I'd rather use short lived. But it's complicated. Need that GitHub OIDC integration.

Reply:
I have created a separate [commit](https://github.com/sunodo/sunodo/pull/49/commits/00d261a830486ba3f4789228c30a0e4d7a359baa) to use terraform module to create the oclif user for us, if it's not needed, I can remove it .

Reply:
For reference, this is what I was talking about
https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services

Reply:
We use that strategy to run `cdk [diff|deploy]` via GHA on another private repository.

Reply:
> https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services

we need to set credentials  in `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment vars The question is: 
  does the oclif work with this option ?

Reply:
It does, because what it does it to generate short-lived credentials with AWS STS, and set the vars.

Reply:
Fixed !